backends/bmv2/README.md

```diff
@@ -17,3 +17,30 @@ To run and test this back-end you need some additional tools:
 - the Python scapy library for manipulating network packets `sudo pip install scapy`
 
 - the Python ipaddr library `sudo pip install ipaddr`
+
+# Unsupported P4_16 language features
+
+Here are some unsupported features we are aware of. We will update this list as
+more features get supported in the bmv2 compiler backend and as we discover more
+issues.
+
+- nested structs:
+```
+struct s0_t {
+  bit<8> f1;
+  bit<8> f2;
+};
+
+struct s1_t {
+  s1_t s01;
+  s1_t s02;
+};
+```
+
```

Comment:
Headers can't contain nested structs, but it seems to me that we could support nested structs in the "header type" that we instantiate the package with. For `v1model.p4`, that's the `H` type parameter to `package V1Switch<H,M>`. I'd expect to be able to do something like this:

```p4
header h1 {...}
struct s1 { h1 h; }
header h2 {...}
struct s2 {h2 h; }

// This type contains nested structs:
struct Headers { s1 nested1; s2 nested2; }

// This should work, but last I checked, it doesn't:
parser p(packet_in b, out Headers h, inout Meta m, inout standard_metadata_t sm) { ... }
```